Doc/whatsnew/3.14.rst

```diff
@@ -113,6 +113,8 @@ Removed
   These had previously raised a :exc:`DeprecationWarning` since Python 3.12.
   (Contributed by Raymond Hettinger in :gh:`101588`.)
 
+* Remove deprecated ``pty.master_open`` and ``pty.slave_open``.
+  They had previously raised a :exc:`DeprecationWarning` since Python 3.12.
```

Comment:
Please can you move this into a "pty" section, above the "Others" I added in https://github.com/python/cpython/pull/118799? Let's list the modules alphabetically like in 3.13.